sentry/src/test/java/io/sentry/HubTest.kt

```diff
@@ -324,6 +324,22 @@ class HubTest {
         verify(mockClient).captureEvent(eq(event), any(), eq(hint))
     }
 
+    @Test
+    fun `when captureEvent is called and event has exception which root cause has been previously attached with span context, sets span context to the event`() {
```

Comment:
We're missing a test where we called `setThrowable` passing a throwable that itself wasn't the root one

Reply:
Got it.